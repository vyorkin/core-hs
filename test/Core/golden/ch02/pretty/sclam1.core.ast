[(Name "f",[Name "x"],ELam [Name "y"] (EAp (EVar (Name "y")) (EVar (Name "x"))))]
