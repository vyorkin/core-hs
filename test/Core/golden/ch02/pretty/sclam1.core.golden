f x = \ y. y x